README.md

```diff
@@ -20,13 +25,13 @@ Requirements:
 
 - Node 6+.
 
-Install Underreact as a dependency of your project:
+Install Underreact as a developer dependency of your project:
 
 ```
-npm install @mapbox/underreact
+npm install -D @mapbox/underreact
```

Comment:
For documentation purposes it's often nice to use the more explicit version, `--save-dev`. (I, for one, am not accustomed to using the shorthand. I should start!)